guides/players/radios.md

```diff
@@ -48,7 +48,7 @@ When interacting with ACRE radios there are two ways to bring up their interface
 <hr/>
 
 # AN/PRC-343 Usage
-The AN/PRC-343 is a short range intra-team level radio commonly carried and used by members within a team to communicate with one another. 
+The AN/PRC-343 is a short range intrateam radio commonly carried and used by members within a team to communicate with one another. 

```

Comment:
should be either `intra-team` or `team`

Reply:
Should level be readded?

Reply:
Remove level from all of them

Reply:
Nevermind, we'll just go with team, its the obvious choice cant believe I didnt think of that from the start

Reply:
It should probably be `rifleman`, and the 148 can the just be `team`